ee/admin/backup/back-up-ucp.md

```diff
@@ -44,10 +46,12 @@ Backup contents are stored in a `.tar` file. Backups contain UCP configuration m
 
 ## Kubernetes settings, data, and state
 
-UCP backups include all kubernetes declarative objects (pods, deployments, replicasets, configs...), including secrets.
+UCP backups include all Kubernetes declarative objects (pods, deployments, replicasets, configurations, and so on), including secrets. These objects are stored in the `ucp-kv etcd` database that is backed up (and restored) as part of UCP backup/restore.
 
-> **Note**: Kube volumes and kube node labels are not be backed up. 
-Upon restore, kubernetes declarative objects are re-created. Containers are re-created and IPs are resolved. 
+
+> Note
+>
+> Kubernetes volumes and node labels are not able to be backed up. Instead, upon restore, Kubernetes declarative objects are re-created. Containers are re-created and IP addresses are resolved. 
```

Comment:
Can rewrite this is active voice for clarity? Something on the lines of:

You cannot back up Kubernetes volumes and node labels.